src/Init/Data/Int/Bitwise/Lemmas.lean

```diff
@@ -0,0 +1,59 @@
+/-
+Copyright (c) 2023 Siddharth Bhat. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Siddharth Bhat, Jeremy Avigad
+-/
+prelude
+import Init.Data.Nat.Bitwise.Lemmas
+import Init.Data.Int.Bitwise
+
+namespace Int
+
+theorem shiftRight_eq (n : Int) (s : Nat) : n >>> s = Int.shiftRight n s := rfl
+@[simp]
+theorem shiftRight_ofNat (n s : Nat) : (n : Int) >>> s = Int.ofNat (n >>> s) := rfl
```

Comment:
The LHS is identical to the LHS of `natCast_shiftRight`, and the RHS unnecessarily introduces non-simp-normal-form. Please delete and use `natCast_shiftRight` instead.

Reply:
Done. Switched to `natCast_shiftRight`